dask/dataframe/io/parquet.py

```diff
@@ -74,6 +74,11 @@ def _read_fastparquet(fs, paths, myopen, columns=None, filters=None,
     elif len(minmax) > 1:
         if index:
             index_col = index
+        elif index is None:
+            # attempt auto-index
+            index = pf._get_index()
+            if index:
+                index_col = index
         elif 'index' in minmax:
             index_col = 'index'
```

Comment:
OK, fair enough, I take your point.
I'll change this soon. (there are on-going problems with the new release of pytest, causing the travis failures, I'll wait for those and merge them in)

Hopefully we can later think of a way to optimize filters on the generated dask-dataframe to be able to avoid loading of some partitions, which currently only works on the index; then this matter would be moot anyway.